docs/user/archives.md

```diff
@@ -0,0 +1,59 @@
+# Archives
```

Comment:
question: would this be clearer as `Archive Formats`, or even `File Formats`, `Artifacts` or something else to distance the concept from Project Archival?
I like the short-one-name for shorter URLs, but fear it may add confusion.